bundle/src/main/java/com/adobe/acs/commons/ondeploy/impl/OnDeployExecutorImpl.java

```diff
@@ -197,12 +200,12 @@ protected boolean runScript(ResourceResolver resourceResolver, OnDeployScript sc
             try {
                 script.execute(resourceResolver);
                 logger.info("On-deploy script completed successfully: {}", statusResource.getPath());
-                trackScriptEnd(statusResource, SCRIPT_STATUS_SUCCESS);
+                trackScriptEnd(statusResource, SCRIPT_STATUS_SUCCESS, null);
```

Comment:
Should `trackScriptStart` be updated to null out the `output` property on the status node as well?

Reply:
I only expect output when completing the script (successful or failure).

Reply:
Right, but if you dont clear the status out at the beginning of the run then this scenario exists:

- Script A fails and writes output stacktrace of "it broke"
- Script A sits with status "failure" and output "it broke" for awhile
- Code (or deployment) triggers Script A to run again, setting status back to "running"
- _Script A has a status of "running" and output of "it broke" until it finishes_
- Script A finishes successfully and clears out the output property

It's item 4 that I'm concerned with, since the old output remains while the new script instance is running.  Since there is only one status node, the status node should remain consistent with a single run - thus when it's running the "output" should be empty.

Reply:
That's indeed a valid use case that I missed.
I implemented the suggested approach. Please review.